products/extensions/b2b-components/employee-management/concept/entities-and-schema.md

```diff
@@ -0,0 +1,47 @@
+# Entities and schema
+
+## Entities
+
+### Company
+
+The company entity represents a company that is registered as a customer in the shop. It is used to gather a company's employees and roles.
```

Comment:
I think "The company entity represents a company that is registered as a customer in the shop" is a bit unclear. Maybe something like "The company entity includes additional B2B company data and extends the basic shop customer"?

Reply:
I changed it with using your and meyers suggestions. Please recheck after push